### ISSUE TITLE
Could u please share the config file for STL?

### ISSUE THREAD

**Issue by kasteric** [Hzzone/ProPos#11]
Hi, I have spent a  really tough time trying to reimplement BYOL on STL, would u mind sharing the config file for STL, my reproduced performance below only reaches around 60%, which gets stuck at 200 epochs...
![image](https://user-images.githubusercontent.com/132050661/236963224-83920f88-b8ef-4396-b49d-993bceb3ee76.png)

**Comment by kasteric** [comment]
The unlabeled and labeled byol loss seem to get no lower after several epochs...
![image](https://user-images.githubusercontent.com/132050661/236963793-8187ecbe-aa15-4ed0-b73a-004284bef3b7.png)

**Comment by Hzzone** [comment]
I do not know what's wrong with you.
Here are the config and logs for BYOL on STL10 with image size 96.
<img width="548" alt="image" src="https://user-images.githubusercontent.com/19267349/236993866-ddf05235-80a8-4924-a993-a2809394d7cf.png">
```
{
    "seed": {
        "desc": null,
        "value": 2
    },
    "_wandb": {
        "desc": null,
        "value": {
            "framework": "torch",
            "cli_version": "0.10.8",
            "is_jupyter_run": true,
            "python_version": "3.7.10",
            "is_kaggle_kernel": false
        }
    },
    "epochs": {
        "desc": null,
        "value": 1000
    },
    "dataset": {
        "desc": null,
        "value": "stl10"
    },
    "feat_dim": {
        "desc": null,
        "value": 256
    },
    "img_size": {
        "desc": null,
        "value": 96
    },
    "momentum": {
        "desc": null,
        "value": 0.9
    },
    "reassign": {
        "desc": null,
        "value": 10
    },
    "run_name": {
        "desc": null,
        "value": "stl10_org_byol_resnet34_3"
    },
    "use_copy": {
        "desc": null,
        "value": 1
    },
    "save_freq": {
        "desc": null,
        "value": 50
    },
    "symmetric": {
        "desc": null,
        "value": 1
    },
    "batch_size": {
        "desc": null,
        "value": 64
    },
    "latent_std": {
        "desc": null,
        "value": 0
    },
    "local_rank": {
        "desc": null,
        "value": 0
    },
    "model_name": {
        "desc": null,
        "value": "offline_contrast"
    },
    "print_freq": {
        "desc": null,
        "value": 10
    },
    "data_folder": {
        "desc": null,
        "value": "/apdcephfs/private_freyaxiong/ft_local/STL10"
    },
    "hidden_size": {
        "desc": null,
        "value": 4096
    },
    "num_workers": {
        "desc": null,
        "value": 32
    },
    "temperature": {
        "desc": null,
        "value": 0.5
    },
    "encoder_name": {
        "desc": null,
        "value": "resnet34"
    },
    "resume_epoch": {
        "desc": null,
        "value": 0
    },
    "shuffling_bn": {
        "desc": null,
        "value": 1
    },
    "weight_decay": {
        "desc": null,
        "value": 0.0005
    },
    "byol_momentum": {
        "desc": null,
        "value": 0.996
    },
    "data_resample": {
        "desc": null,
        "value": 0
    },
    "learning_rate": {
        "desc": null,
        "value": 0.05
    },
    "warmup_epochs": {
        "desc": null,
        "value": 50
    },
    "whole_dataset": {
        "desc": null,
        "value": 1
    },
    "fix_predictor_lr": {
        "desc": null,
        "value": 0
    },
    "use_gaussian_blur": {
        "desc": null,
        "value": 0
    },
    "cluster_loss_weight": {
        "desc": null,
        "value": 0
    },
    "lambda_predictor_lr": {
        "desc": null,
        "value": 10
    }
}
```
The accuracy is ~83%, reaching the highest at ~800th epoch.
